sunpy/map/mapbase.py

```diff
@@ -1787,6 +1797,13 @@ def draw_rectangle(self, bottom_left, *, width: u.deg = None, height: u.deg = No
         """
         Draw a rectangle defined in world coordinates on the plot.
 
+        This draws a rectangle that has corners at ``(bottom_left, top_right)``,
+        and has sides parallel to the x and y axes of the plot.
```

Comment:
This wording is potentially confusing. They have axes aligned with the coordinate grid not the axes of the plot. I think they might be best described as lines of constant latitude / longitude.

Reply:
> This wording is potentially confusing.

I think you meant "wrong" 😆 . Will fix.

Reply:
I think latitude / longitude is potentially confusing (for a helioprojective map it might be interpreted as lines of Carrington lat/long), so I just went with coordinate axes, which is still a bit vague but I think true.